src/_includes/partials/components/pagination.njk

```diff
@@ -0,0 +1,13 @@
+<nav class="pagination" aria-label="pagination">
```

Comment:
`pagination.njk` doesn't seem to be used anywhere. Can you double check? If it doesn't, please remove. Thanks.